packages/app/package.json

```diff
@@ -191,6 +191,7 @@
     "instantsearch.css": "^7.1.0",
     "is-url": "^1.2.2",
     "jest-circus": "^22.1.4",
+    "jest-each": "^23.6.0",
```

Comment:
You might want to use the latest version of [`jest-each@24.8.0`](https://www.npmjs.com/package/jest-each/v/24.8.0), there has been a few bug fixes etc since 23.6.0

Reply:
That makes sense to me! Other than that this PR is really good, thanks a lot @batiskafff. It greatly improves the quality of our jest integration.

Reply:
Hi @mattphillips @CompuIves 
I have tried to use jest-each@24.x.x, however it doesn't work. It fail with compilation error. I didn't tackle this problem and decided to go with 23.6.0

I will come back to it and give more detail about this issue.

Reply:
@batiskafff When you do, please also rebase this to master, so that it gets picked up by the new PR build system. Thanks!

Reply:
done